play-media-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "play-media-native",
-  "version": "1.0.0",
```

Comment:
I think we still need the version attribute.
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#version

Reply:
Version attribute is optional for our case (mandatory only when publishing to npm). We could put it back but I think it will be kinda confusing having both baseVersion and version in there.